rust/abacus-base/src/oneline_eyre/handler.rs

```diff
@@ -86,6 +86,7 @@ pub struct HookBuilder {
     show_full_paths: bool,
 }
 
+#[allow(dead_code)]
```

Comment:
why?

Reply:
Because it is a builder, not using a function now does not mean it should not be there.